src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -321,6 +321,13 @@ public static FeedbackResponseCommentSearchResultBundle fromResults(
             bundle.commentGiverTable.put(comment.getId().toString(),
                     getFilteredCommentGiverName(bundle, instructorCourseIdList, response, comment, commentGiverName));
             bundle.instructorEmailNameTable.put(comment.giverEmail, commentGiverName);
+            if (!comment.lastEditorEmail.isEmpty()
+                    && !bundle.instructorEmailNameTable.containsKey(comment.lastEditorEmail)) {
+                InstructorAttributes instructor = instructorsDb.getInstructorForEmail(response.courseId,
+                        comment.lastEditorEmail);
+                String commentLastEditorName = instructor.displayedName + " " + instructor.name;
+                bundle.instructorEmailNameTable.put(comment.lastEditorEmail, commentLastEditorName);
+            }
             bundle.numberOfResults++;
```

Comment:
Yes, we can add a test to cover this scenario. I'll update the PR with requested changes.